spec/unit/patterns_inspector_spec.rb

```diff
@@ -55,11 +71,9 @@
   }
 
   describe "#inspect" do
-    before(:each) do
-      allow(system).to receive(:has_command?).with("zypper").and_return(true)
-    end
 
-    it "parses the patterns list into a Hash" do
+    it "parses the patterns list into a Hash on a zypper based OS" do
+      allow(system).to receive(:has_command?).with("zypper").and_return(true)
```

Comment:
I see a bunch of repetition with this call. I think it'd look nicer if there was a context separation for "zypper" and "taskel" and they both would have a `before(:each)` with the according setup.